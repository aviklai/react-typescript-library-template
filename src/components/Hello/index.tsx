import * as React from 'react';

export interface Props {
    text: string
}

export class Hello extends React.Component<Props> {
  constructor(props: Props, context?: any) {
    super(props, context);  
  }

  render() {   
    return (
        <div className='hello'>
            {this.props.text}
        </div>
    ); 
  }
} 