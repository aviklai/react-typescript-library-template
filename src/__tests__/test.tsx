import * as React from 'react';
import * as renderer from 'react-test-renderer';
import { Hello } from '../components/Hello/index';

import '../components/Hello/styles.scss';

it('Hello renders correctly', () => {
  const tree = renderer
    .create(<Hello text="hello" />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});