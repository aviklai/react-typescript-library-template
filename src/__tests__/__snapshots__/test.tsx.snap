// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Hello renders correctly 1`] = `
<div
  className="hello"
>
  hello
</div>
`;
